integration_tests/src/main/python/map_test.py

```diff
@@ -322,9 +322,17 @@ def test_simple_get_map_value_ansi_fail(data_gen):
             conf=ansi_enabled_conf,
             error_message=message)
 
+@pytest.mark.skipif(is_before_spark_340(),
```

Comment:
Is the `strict_index` missing in this tests ?

Reply:
The config `spark.sql.ansi.strictIndexOperator` has been removed in Spark340, see https://github.com/apache/spark/commit/95cbd61857